src/main/java/net/neoforged/neoforge/capabilities/CapabilityHooks.java

```diff
@@ -47,6 +50,10 @@ public static void init() {
 
         var event = new RegisterCapabilitiesEvent();
         ModLoader.postEventWrapContainerInModOrder(event);
+        var nonProxyableCapabilities = BlockCapability.getAllProxyable().stream().filter(c -> !c.proxyable).toList();
```

Comment:
I believe there is a small mistake here and this should be `BlockCapability#getAll` rather than `BlockCapability#getAllProxyable` as `#getAllProxyable` would only return those capabilities for which `proxyable` is true.

Reply:
Oops, thanks! (Now this code is gone in the latest version.)